README.md

```diff
@@ -52,7 +52,7 @@ Live Community Chat: [![Gitter](https://img.shields.io/badge/gitter-join_chat-bl
 
 ### Building NuPIC From Source Code
 
-For details about checking out this repository and building in your local environment, see the [Installing and Building NuPIC](https://github.com/numenta/nupic/wiki/Installing-and-Building-NuPIC) wiki page.
+For details about checking out this repository and building in your local environment, see the [Installing and Building NuPIC](https://discourse.numenta.org/t/building-nupic/2192) wiki page.
```

Comment:
I think we can replace this with the command to install from repo, which is the same as the current install command but replacing "nupic" with "." when you're in the repo directory:

```
pip install .
```

Optionally, `-e` can be specified for a developer install.